SimpleInventoryManagementSystem/Product.cs

```diff
@@ -1,19 +1,17 @@
-﻿
-using System;
+﻿using System;
 
-namespace ISimpleInventoryManagementSystem
+namespace SimpleInventoryManagementSystem;
+
+public class Product
 {
-    public class Product
-    {
-        public string Name { get; set; }
-        public decimal Price { get; set; }
-        public int Quantity { get; set; }
+    public string Name { get; set; }
+    public float Price { get; set; }
+    public int Quantity { get; set; }
```

Comment:
```suggestion
    public required string Name { get; set; }
    public required float Price { get; set; }
    public required int Quantity { get; set; }
```
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions
![image](https://github.com/user-attachments/assets/be696279-d646-402e-8792-9504a3a615fc)